tasks/cron.yml

```diff
@@ -0,0 +1,8 @@
+- name: Update topologies cron
+  become: yes
```

Comment:
change this to `true` for consistency with the rest of boolean values used in the playbook